modules/EnsEMBL/Draw/GlyphSet/_mane_select.pm

```diff
@@ -0,0 +1,35 @@
+=head1 LICENSE
+
+Copyright [1999-2015] Wellcome Trust Sanger Institute and the EMBL-European Bioinformatics Institute
+Copyright [2016-2021] EMBL-European Bioinformatics Institute
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+     http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+
+=cut
+
+package EnsEMBL::Draw::GlyphSet::_mane_select;
+
+### Module for drawing the MANE Select track inheriting from _transcript.pm.
+
+use strict;
+use Carp qw(cluck);
```

Comment:
same here, the 2 import are not needed and can be removed.

Reply:
That's done